### DISCUSSION TITLE
How to choose which badges to display ?

### DISCUSSION THREAD

**Discussion by aaronlamz** [community/community#20536] [Other Feature Feedback, Questions, & Ideas] [answered]
How to choose which badges to display

**Comment by melvincwng** [comment, marked as answer]
Hi @aaronlamz, as of right now, it's not possible to selectively choose which badges/achievements to be displayed.

You can refer to [this link](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/managing-contribution-settings-on-your-profile/showing-your-private-contributions-and-achievements-on-your-profile) if you wish to hide **all** your achievement badges.

Hope this clarifies.

**Reply by aaronlamz** [reply]
got it. thanks for the information.
